php/Blockonomics.php

```diff
@@ -238,4 +294,334 @@ public function testSetup()
+        if($this->is_lite_mode_active()){
+        ?>
+          <link rel="stylesheet" type="text/css" href="<?php echo plugins_url('css/order.css', dirname(__FILE__));?>">
+        <?php
+        }else{
+          get_header();
+        }
+    }
+
+    // Adds the footer to the blockonomics page
+    public function load_blockonomics_footer($template_name){
+        // Lite mode will render without wordpress theme footers
+        if($this->is_lite_mode_active()){
+            // Only load the lite mode javascript if nojs is not active
+            if (!$this->is_nojs_template($template_name)) {
+                ?>
+                  <script>var ajax_object = {ajax_url:"<?php echo admin_url( 'admin-ajax.php' ); ?>", wc_url:"<?php echo WC()->api_request_url('WC_Gateway_Blockonomics'); ?>"};
+                  </script>
+                  <script src="<?php echo plugins_url('js/angular.min.js', dirname(__FILE__));?>"></script>
+                  <script src="<?php echo plugins_url('js/angular-resource.min.js', dirname(__FILE__));?>"></script>
+                  <script src="<?php echo plugins_url('js/app.js', dirname(__FILE__));?>"></script>
+                  <script src="<?php echo plugins_url('js/angular-qrcode.js', dirname(__FILE__));?>"></script>
+                  <script src="<?php echo plugins_url('js/vendors.min.js', dirname(__FILE__));?>"></script>
+                  <script src="<?php echo plugins_url('js/reconnecting-websocket.min.js', dirname(__FILE__));?>"></script>
+                <?php
+            }
+        }else{
+          get_footer();
+        }
+    }
+
+    // Adds the selected template to the blockonomics page
+    public function load_blockonomics_template($template_name){
+        $this->load_blockonomics_header($template_name);
+
+        // Load the selected template
+        // Check if child theme or parent theme have overridden the template
+        $template = 'blockonomics_'.$template_name.'.php';
+        if ( $overridden_template = locate_template( $template ) ) {
+            load_template( $overridden_template );
+        } else {
+            load_template( plugin_dir_path(__FILE__)."../templates/" .$template );
+        }
+
+        $this->load_blockonomics_footer($template_name);
+
+        exit();
+    }
+
+    // Load the the payment confirmed template in the page
+    public function load_nojs_payment_confirmation_template(){
+        $this->load_blockonomics_template('nojs_payment_confirmation');
+    }
+
+    // Check if any pending payments linked to the order
+    public function check_for_pending_payment($orders, $order_id){
+        $network_confirmations = get_option("blockonomics_network_confirmation",2);
+        foreach ($orders as $address => $order){
+            if ($order['status'] >= 0 && $order['status'] < $network_confirmations){
+                $this->redirect_finish_order($show_order);
+            };
+        };
+    }
+
+    public function update_waiting_order($order){
```

Comment:
Rename to *calculate_order_params* as it is more appropriate

Reply:
Renamed in 429b6c6